crates/astria-sequencer/src/grpc/sequencer.rs

```diff
@@ -117,15 +118,22 @@ impl SequencerService for SequencerServer {
                 ))
             })?;
 
-        let (rollup_transactions_proof, rollup_ids_proof) = snapshot
-            .get_block_proofs_by_block_hash(&block_hash)
+        let rollup_transactions_proof = snapshot
+            .get_rollup_transactions_proof_by_block_hash(&block_hash)
             .await
             .map_err(|e| {
                 Status::internal(format!(
-                    "failed to get sequencer block proofs from storage: {e}"
+                    "failed to get rollup transactions proof from storage: {e}"
```

Comment:
Same as below: it's technically out of scope for this PR, but I haven't noticed this earlier. Let's report the error as `{e:#}`?

Reply:
As per reply above.